src/librustdoc/clean/types.rs

```diff
@@ -1067,9 +1067,6 @@ impl Attributes {
         let mut out = String::new();
         add_doc_fragment(&mut out, ori);
         for new_frag in iter {
-            if new_frag.kind != ori.kind || new_frag.parent_module != ori.parent_module {
-                break;
-            }
```

Comment:
nit: this means that the `iter.next()` and such seems no longer necessary here (all elements of the iterator are directly added), but if you're just leaving that for easier revert seems OK.

Reply:
Let's see what @camelid thinks about it.

Reply:
That's a good point, but I do want to make this easy to revert quickly.